src/metabase/driver/mysql.clj

```diff
@@ -65,7 +65,8 @@
       kdb/mysql
       ;; 0000-00-00 dates are valid in MySQL, but JDBC barfs when queries return them because java.sql.Date doesn't allow it.
       ;; Add a param to the end of the connection string that tells MySQL to convert 0000-00-00 dates to NULL when returning them.
-      (update :subname (u/rpartial str "?zeroDateTimeBehavior=convertToNull"))))
+      ;; Also add params to force UTF-8 encoding of results
+      (update :subname (u/rpartial str "?zeroDateTimeBehavior=convertToNull&useUnicode=true&characterEncoding=UTF8&characterSetResults=UTF8"))))
```

Comment:
I agree 100% that using utf-8 everywhere is ideal, i'm simply pointing out that we can't control the choices our users may have already made and thus we should be as accommodating as possible.  

If someone shows up with a MySQL database that isn't using utf-8 for the data then this could make the app unusable for them, which i'd like to avoid.